lib/discordrb/data/role.rb

```diff
@@ -35,6 +35,39 @@ class Role
     # @return [String, nil] The icon hash for this role.
     attr_reader :icon
 
+    # @return [Tags, nil] The role tags
+    attr_reader :tags
+
+    # Wrapper for the role tags
+    class Tags
+      # @return [Integer, nil] The ID of the bot this role belongs to
+      attr_reader :bot_id
+
+      # @return [Integer, nil] The ID of the integration this role belongs to
+      attr_reader :integration_id
+
+      # @return [true, false] Whether this is the guild's Booster role
+      attr_reader :premium_subscriber
+
+      # @return [Integer, nil] The id of this role's subscription sku and listing
+      attr_reader :subscription_listing_id
+
+      # @return [true, false] Whether this role is available for purchase
+      attr_reader :available_for_purchase
+
+      # @return [true, false] Whether this role is a guild's linked role
+      attr_reader :guild_connections
+
+      def initialize(data)
+        @bot_id = data['bot_id']
```

Comment:
```suggestion
        @bot_id = data['bot_id']&.resolve_id
```
To be consistent with your documentation, which indicates that you return Integer